ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -608,13 +608,20 @@ public class RouterConfig {
   public final long routerSmallestBlobForMetadataCache;
   public static final long NUM_BYTES_IN_ONE_TB = (long) Math.pow(1024, 4);
 
+  public static final String ROUTER_MAX_NUM_METADATA_CACHE_ENTRIES = "router.max.num.metadata.cache.entries";
+  @Config(ROUTER_MAX_NUM_METADATA_CACHE_ENTRIES)
+  public final int routerMaxNumMetadataCacheEntries;
+  public static final int MAX_NUM_METADATA_CACHE_ENTRIES = 10;
```

Comment:
Let's make the name consistent.  Suggest changing name to MAX_NUM_METADATA_CACHE_ENTRIES_DEFAULT

Reply:
thanks